config/routes.rb

```diff
@@ -1,5 +1,5 @@
 MANIACChess::Application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```

Comment:
Right here, you're listing the callback controller as part of the Users namespace but you don't actually have it set up that way, so I suspect the routing will fail.